### DISCUSSION TITLE
Connection error

### DISCUSSION THREAD

**Discussion by saukkico** [rustdesk/rustdesk#9505] [General]
### Bug Description

About every 30 seconds the connection between my local and remote computers freeze. Then it stays frozen about 20 to 30 seconds and then I get a error message: Connection error. Operation timed out (os error 60)
If I connect from my iPhone to the same remote computer I have no issues.

### How to Reproduce

Connect from local Apple silicon Mac to remote Intel Mac, wait about 30 seconds for the connection to freeze

### Expected Behavior

Connection should work normally without any freezing.

### Operating system(s) on local side and remote side

macOS 14.7 (Apple silicon MBP) -> macOS 14.7 (Intel Macmini)

### RustDesk Version(s) on local side and remote side

Just updated both to latest: Version 1.3.1 (47)

### Screenshots

<img width="369" alt="Näyttökuva 2024-09-28 kello 13 06 04" src="https://github.com/user-attachments/assets/4b57a6fe-9b6d-482c-a199-0f4dc22997df">

### Additional Context

Above error message shows for about 2 seconds after which the connection works for another 30 seconds before it freezes again. I'm running my own Rustdesk server. It should be on the latest version as well. I'm pretty sure the server is working ok, because the connection from my iPhone to the remote Macmini works just fine. The issue only occurs when connecting from my Macbook Pro to the remote Mac. IPhone and the local Macbook Pro are on the same wifi.

**Comment by saukkico** [comment]
Any ideas why this is happening? Anyone?

**Comment by tomasz1986** [comment]
Same problem here but between Windows machines. This only happens when connecting to one of them, the rest all seem to work fine. I've tried using the nightly versions but to no avail.

**Comment by saukkico** [comment]
I also run a Rustdesk client in Kasm (using this image: https://github.com/JReming85/rustdesk-kasm-image) and that also seems to work just fine when connecting to the Macmini. So weirdly it's only the client on my MBP which is causing problems.

**Comment by saukkico** [comment]
Turns out this is not a Rustdesk issue at all. The problem has something to do with my Twingate client. After closing the Twingate connection, the Mac's network is somehow unstable. I haven't yet figured out what exactly is going on. But I figured if I briefly turn on  and off some other VPN client, then the network works without any issue. So it seems that the Twingate client somehow breaks the Mac's networking, and that's why I get this connection error every 30 seconds. I get similar connection issues with some other apps as well.

**Comment by mercgarage** [comment]
Stop RustDesk client service or uninstall the client
Delete folders under 
```
C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\
%AppData%\
```
Start services/Install client will solve the issue.
